### PR TITLE
Create slso.md

### PR DESCRIPTION
**Pull request by DanBerrios**
This is the new metadata record for the new ontology Space Life Sciences Ontology (SLSO) as requested in  #2518 .  Note in the proposed metadata, I omitted the "mirror_from" attribute which was pointing to a working branch in the SLSO repository, since I have now merged all changes recommended and reviewed by OBO Operations on that branch into the SLSO master branch.

### CODE REVIEW

**Comment by DanBerrios** [comment]
@jamesaoverton  I corrected an error in the products' URL description....if you rerun the tests, I think they should work now....

**Review by jamesaoverton — APPROVED** [review]
Looks good to me (although I don't review many of these).

File: ontology/slso.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  url: https://bioportal.bioontology.org/ontologies/SLSO
+contact:
+  email: daniel.c.berrios@nasa.gov
+  github: DanBerrios
+  label: Dan Berrios
+  orcid: 0000-0003-4312-9552
+description: The Space Life Sciences Ontology is an application ontology and is intended to support the operation of NASA's Life Sciences Data Archive and other systems that contain space life science research data.
+domain: investigations
+homepage: https://github.com/nasa/LSDAO
+in_foundry_order: 1
```
Comment (matentzn) [inline ontology/slso.md:17]:
```suggestion
```

File: ontology/slso.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+license:
+  label: CC BY 3.0
+  url: http://creativecommons.org/licenses/by/3.0/
```
Comment (matentzn) [inline ontology/slso.md:20]: Not a review comment, @DanBerrios for future ontologies consider CC-BY 4.0 which is a much simpler, less confusing license.

File: ontology/slso.md (hunk 1 above)
Comment (matentzn) [reply ontology/slso.md:17]: Why this statement?

**Review by matentzn — CHANGES_REQUESTED** [review]

File: ontology/slso.md (hunk 1 above)
Comment (DanBerrios) [reply ontology/slso.md:17]: > Hmm. I bet @DanBerrios copied this from another config, like OBI's. It should be removed. Thanks @matentzn.

Yeah I wasn't sure what it meant and copied it from the PATO example...

File: ontology/slso.md (hunk 2 above)
Comment (DanBerrios) [reply ontology/slso.md:20]: I will check the latest guidance from NASA but I believe that last time I checked NASA preferred CC-BY 3.0.

**Review by matentzn — APPROVED** [review]

**Review by deepakunni3 — APPROVED** [review]

**Pull request merged**